src/main/scala/za/co/absa/hyperdrive/trigger/persistance/EventRepository.scala

```diff
@@ -32,15 +32,15 @@ class EventRepositoryImpl @Inject()(val dbProvider: DatabaseProvider) extends Ev
   import api._
 
   override def getAllEvents()(implicit ec: ExecutionContext): Future[Seq[Event]] = db.run(
-    eventTable.result
+    eventTable.result.withErrorHandling()
   )
 
   override def getExistEvents(sensorEventIds: Seq[String])(implicit ec: ExecutionContext): Future[Seq[String]] = db.run(
-    eventTable.filter(e =>  e.sensorEventId.inSet(sensorEventIds)).map(_.sensorEventId).result
+    eventTable.filter(e =>  e.sensorEventId.inSet(sensorEventIds)).map(_.sensorEventId).result.withErrorHandling()
```

Comment:
Sometimes you broke lines, and sometimes you didn't, like here. Would it make sense to properly format this by hand, or is it a useless exercise anyway without a code formatter?

Reply:
For example this line is still reasonably long and fits into 120 chars. That was my reasoning when to broke lines

Reply:
Ah ok, it's about line length. Makes sense